### PR TITLE
fix: fixtures usage example

### PR DESCRIPTION
**Pull request by arianejasuwienas**
**Description**:

Enhance the Waffle usage example by adding additional code that demonstrates how Waffle Fixtures can be utilized.

**Related issue(s)**:

https://github.com/hashgraph/hedera-json-rpc-relay/issues/2882

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by acuarica** [comment]
Hi @arianejasuwienas, thanks for sending this. Could you please fill in the PR description? So we are aware what motivated this change.

File: tools/waffle-example/test/samplecontract.test.mjs [hunk 1]
```diff
@@ -21,31 +21,31 @@
...
 import SampleContract from '../build/SampleContract.json' assert { type: "json" };
 
 dotenv.config();
 const { JsonRpcProvider } = providers;
 
+const provider = new JsonRpcProvider(process.env.RELAY_ENDPOINT);
+const wallet = new Wallet(process.env.OPERATOR_PRIVATE_KEY, provider);
+const loadFixture = createFixtureLoader([wallet]);
+const fixture = ([wallet]) => deployContract(wallet, SampleContract);
+
```
Comment (acuarica) [inline tools/waffle-example/test/samplecontract.test.mjs:35]: This should be move inside a `before` or `beforeEach` block.

**Review by acuarica — CHANGES_REQUESTED** [review]

**Comment by acuarica** [comment]
Hey @arianejasuwienas, there are some outstanding comments. Would you be able to address them so we can merge this?

**Review by acuarica — APPROVED** [review]
lg

**Review by natanasow — APPROVED** [review]
LGTM

**Pull request merged**